### DISCUSSION TITLE
Today I noticed that adguard disables login of the new version of the viaplay app

### DISCUSSION THREAD

**Discussion by adekloet** [hassio-addons/addon-adguard-home#315] [General]
Hi,

Today I noticed that adguard disables login of the new version of the viaplay app. On any device. Turning off adguard fixed it, but ofcourse I just want to know which URL to allow, it seems they use a new service for login or tracking which is blocked. I have trouble finding it in the logs.

I wander if anyone had the same experience and was able to isolate the particular services that are the culprit,

Thanks.

**Comment by adekloet** [comment]
> Hi,
> 
> Today I noticed that adguard disables login of the new version of the viaplay app. On any device. Turning off adguard fixed it, but ofcourse I just want to know which URL to allow, it seems they use a new service for login or tracking which is blocked. I have trouble finding it in the logs.
> 
> I wander if anyone had the same experience and was able to isolate the particular services that are the culprit,
> 
> Thanks.

![Screenshot_20220423-214621_Chrome](https://user-images.githubusercontent.com/60182504/164943988-fd8de428-b68e-4bab-8d14-fe9495f60f2c.jpg)

**Reply by RoRoo1977** [reply]
I don't have ViaPlay, but my SiL was visiting last sunday and I had to disable both my AdGuard servers to get access to ViaPlay. 
I've got: 
AdAway and Dan Pollock in common with you. 

Maybe check the logs when trying to login? It shows which blocklist was being used.

**Comment by Arjuuhn** [comment]
Unblock auth.split.io

**Comment by RandomAK86** [comment]
I got Viaplay-login to work by adding this to custom filter:
@@||auth.split.io^$important
@@||sdk.split.io^$important
